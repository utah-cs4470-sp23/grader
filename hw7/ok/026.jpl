let x = 1
show array[i : x] x
