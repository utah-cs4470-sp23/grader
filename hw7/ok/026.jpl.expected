(LetCmd (ArgLValue (VarArgument x)) (IntExpr (IntType) 1))
(ShowCmd (ArrayLoopExpr (ArrayType (IntType) 1) i (VarExpr (IntType) x) (VarExpr (IntType) x)))